book/common/definitions.tex

```diff
@@ -624,12 +624,13 @@
 
 \begin{SaveDefinition}[key=SimilarMatrices, title={Similar Matrices}]
 	The matrices $A$ and $B$ are called
-	\emph{similar matrices}\index[definitions]{Matrix!similar matrices}, denoted $A\sim B$\index[symbols]{$\sim$}, if $A$ and $B$ represent the
+	\emph{similar matrices}\index[definitions]{Matrix!similar matrices},
+	denoted $A\sim B$\index[symbols]{$\sim$}, if $A$ and $B$ represent the
 	same linear transformation but in possibly different bases. Equivalently,
 	$A\sim B$ if there is an invertible matrix $X$ so that
 	\[
 		A=XBX^{-1}.
-	\]\index[definitions]{Matrix!similar}\index{Matrix!similar}
```

Comment:
These are redundant because `{Matrix!similar matrices}` is already given above on line 627.